static-assets/themes/cstudioTheme/yui/assets/skin.css

```diff
@@ -5644,7 +5644,8 @@ h2.yui-editor-skipheader, h3.yui-editor-skipheader {
   border-bottom-color: #243356;
 }
 
-.ygtvitem {
+.ygtv-highlight0:hover {
+  background-color: #e2e2e2;
```

Comment:
Use rgba(...). Try to mimic material. (We're going to require some theming work legacy side to support dark mode.)